libraries/standard/mqtt/integration-test/mqtt_system_test.c

```diff
@@ -795,23 +817,27 @@ void test_MQTT_Connect_LWT( void )
     TEST_ASSERT_NOT_EQUAL( -1, secondNetworkContext.socketDescriptor );
     TEST_ASSERT_NOT_NULL( secondNetworkContext.pSsl );
 
-    /* Establish MQTT session on top of the TCP+TLS connection. */
+    /*Establish MQTT session on top of the TCP + TLS connection. */
     useLWTClientIdentifier = true;
-    establishMqttSession( &secondContext, &secondNetworkContext, true, &sessionPresent );
+    establishMqttSession( &secondContext, true, &secondNetworkContext, true, &sessionPresent );
 
-    /* Subscribe to LWT Topic. */
+    /*Subscribe to LWT Topic. */
     TEST_ASSERT_EQUAL( MQTTSuccess, subscribeToTopic(
                            &context, TEST_MQTT_LWT_TOPIC, MQTTQoS0 ) );
 
-    /* Abruptly terminate TCP connection. */
+    /*Abruptly terminate TCP connection. */
     ( void ) Openssl_Disconnect( &secondNetworkContext );
 
-    /* Run the process loop to receive the LWT. Allow some more time for the
+    /*Run the process loop to receive the LWT.Allow some more time
+     *
+     * for the
```

Comment:
I think something's wrong with this comment

Reply:
Fixed